src/Symfony/Component/Notifier/Bridge/Telegram/TelegramTransportFactory.php

```diff
@@ -54,4 +53,17 @@ private function getToken(Dsn $dsn): string
 
         return \sprintf('%s:%s', $dsn->getUser(), $dsn->getPassword());
     }
+
+    private function isHttpsDisabled(Dsn $dsn): bool
+    {
+        if (!$disableHttpsOption = $dsn->getOption('disable_https')) {
+            return false;
+        }
+
+        if (!\in_array($disableHttpsOption, [0, 1])) {
```

Comment:
If we use filter_var bool here, one could also use `on` and `off`, which is quite handy. Please also add a test case for this, thanks

Reply:
This is should be done now, you can just have a final check at the tests that I've added, If everything is fine you can resolve this 😀

Reply:
Very good, thanks!